extensions/amp-story-shopping/amp-story-shopping.md

```diff
@@ -0,0 +1,177 @@
+---
+$category@: presentation
+formats:
+  - stories
+teaser:
+  text: A configurable, templated shopping experience in AMP story pages.
+tags:
+  - shopping
+author: philipbell
+toc: true
+draft: true
+$title: amp-story-shopping
+version: '0.1'
+versions:
+  - '0.1'
+latest_version: '0.1'
+$path: /documentation/components/amp-story-shopping.html
+$localization:
+  path: '/{locale}/documentation/components/amp-story-shopping.html'
+scripts:
+  - >-
+    <script async custom-element="amp-story-shopping" src="https://cdn.ampproject.org/v0/amp-story-shopping-0.1.js"></script>
+---
+
+# amp-story-shopping
+
+<amp-img alt="An example of amp story shopping tag in each type of configuration" src="https://user-images.githubusercontent.com/3860311/152416815-61a63a88-76c0-4ae8-9300-a7ad277dbe7a.jpg" layout="intrinsic" width="806" height="428">
+
+The `amp-story-shopping` component creates configurable, templated shopping experiences within [amp-story](https://amp.dev/documentation/components/amp-story/?format=stories).
+
+Specify a shopping experience by defining one `amp-story-shopping-attachment` and one or more `amp-story-shopping-tag` elements for each product on the [`amp-story-page`](https://amp.dev/documentation/components/amp-story-page/?format=stories).
+
+## amp-story-shopping-tag
+
+<amp-img alt="An example of amp story shopping tag in each type of configuration" src="https://user-images.githubusercontent.com/3860311/155754637-3403a9dd-c4c9-44f3-ad6d-e4d166d30ad2.gif" layout="intrinsic" width="844" height="102">
+
+Use `amp-story-shopping-tag` elements to indicate shop-able elements within [`amp-story-page`](https://amp.dev/documentation/components/amp-story-page/?format=stories).
+Tapping them opens a product description page (PDP) within the `amp-story-shopping-attachment`.
+
+-   They must be a child of [`amp-story-grid-layer`](https://amp.dev/documentation/components/amp-story-grid-layer/).
```

Comment:
This list needs an introduction of what it is a list of. Maybe something like: `To use `amp-story-shopping-tag` it must meet the following requirements:` - and then remove the `They must`/`they are` from the first two points.